MIGRATION.md

```diff
@@ -0,0 +1,5 @@
+# Migration Guide
+
+This document provides instructions for migrating your codebase to accommodate breaking changes introduced in new versions of Griptape.
+
+## 0.31.0
```

Comment:
should the heading be something like `0.30 -> 0.31`? i know this is the first one but it needs to delta off of something.

Reply:
Done